.github/workflows/elixir.yml

```diff
@@ -20,13 +19,36 @@ jobs:
         otp-version: 22.2 # Define the OTP version [required]
     - name: Check formatting
       run: mix format --check-formatted
+
+    - name: Cache Dependencies
+      id: mix-cache
+      uses: actions/cache@v2
+      with:
+        path: deps
+        key: ${{ runner.os }}-mix-${{ hashFiles(format('{0}{1}', github.workspace, '/mix.lock')) }}
+        restore-keys: |
+          ${{ runner.os }}-mix-
```

Comment:
I don't completely understand the intention behind the restore-keys - when a key begins with "${{ runner.os }}-mix-" in this case, will it then be checked again against the actual "key"?

Reply:
Of I undersrood right, then restore-keys is a list of key(-parts) that will be used to restore an earlier cache in case the implicitely defined key ist not found. See https://help.github.com/en/actions/configuring-and-managing-workflows/caching-dependencies-to-speed-up-workflows#matching-a-cache-key

Not sure, maybe we don't need it here actually...

Reply:
It seems to me that if the exact key is not found it will look for caches with not exact matches (starting with ${{ runner.os }}-mix-) - I guess this is not what we went and we should remove the restore-keys line

Reply:
I removed the `restore-keys` keys.